### ISSUE TITLE
What happened to GitCommits?

### ISSUE THREAD

**Issue by pauldotknopf** [statiqdev/Statiq#648]

**Comment by LokiMidgard** [comment]
I think this module was removed due to incompability with .NetCore.

**Comment by pauldotknopf** [comment]
The c# git interop stuff now supports .NET Core.

Also, I doubt it was removed because of that, considering .NET Core hasn't been implemented yet, and there are many other dependencies that only target full framework.

**Comment by LokiMidgard** [comment]
It was removed with version [1.1.0](https://github.com/Wyamio/Wyam/releases/tag/v1.1.0). It was removed for preperations for a cross platform variation (as far as I kown, see [xslt2 module](#340)).
I think it was one of the first steps to .NetCore. 

I had planed to create an external package that will not be platform indepandent. But I havent worked on my Project that uses this since a long time. So I haven't updated Wyam since then. It still runs on an version before 1.1.0 with git and XSLT 2.0 Support.

**Comment by daveaglick** [comment]
Yep, @LokiMidgard has it about right. The packages that enabled this module included unmanaged libraries but didn't pack them in the NuGet package correctly for the new SDK. When I made the switch for the projects to the new SDK format I decided not to spend too much time on this module since it was already lightly used and removed it.

**Comment by pauldotknopf** [comment]
0.25 supports netstandard: https://www.nuget.org/packages/LibGit2Sharp/0.25.0-preview-0081

**Comment by daveaglick** [comment]
Interesting, might be worth looking at bringing it back then. We'd need to be careful though since it looks like 0.25 is still pre-release. NuGet enforces transitive pre-release so the module package would also need to be marked as pre-release if we brought it back at this point.
